### PR TITLE
Exercise 011 Task 1,2,3,4 Solution Patertuck

### PR DESCRIPTION
**Pull request by Patertuck**
Also added Test file for 11.1

### CODE REVIEW

File: Informatics I/011 Word Game and Comprehensions/Task 1 List and Dict Comprehensions/Patertuck Solution/script.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+from data import words
+import string
+
+def words_with_length(length):
+    '''this one just serves as an example'''
+    return [word for word in words if len(word) == length]
+
+def words_containing_string(s):
+    return [s_word for s_word in words if s in s_word]
```
Comment (C0DECYCLE) [inline Informatics I/011 Word Game and Comprehensions/Task 1 List and Dict Comprehensions/Patertuck Solution/script.py:11]: nice and clean!

File: Informatics I/011 Word Game and Comprehensions/Task 2 Hex-Code Generator/Patertuck Solution/script.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+from random import choice
+
+class GameRunner(object):
+
+    def __init__(self):
+        self.rows = 17
+        self.columns = 2
+
+    def generate_hex_codes(self):
+        return [("0x" + "".join([choice("0123456789ABCDEF") for _ in range(4)]))for _ in range (self.columns * self.rows)]
```
Comment (C0DECYCLE) [inline Informatics I/011 Word Game and Comprehensions/Task 2 Hex-Code Generator/Patertuck Solution/script.py:11]: very nice! (minor thing: missing space before "for" and additional after "range")

**Review by C0DECYCLE — APPROVED** [review]
Intersting solutions, i like it! Thank you very much🤝
(style guide tip: in clases sort methods and attributes by, public - protected - private)

Edit: And thank you very much for your tests!

File: Informatics I/011 Word Game and Comprehensions/Task 2 Hex-Code Generator/Patertuck Solution/script.py (hunk 2 above)
Comment (Patertuck) [reply Informatics I/011 Word Game and Comprehensions/Task 2 Hex-Code Generator/Patertuck Solution/script.py:11]: ah yes, is this a change that is worth me reopening the branch and fixing it?

**Comment by Patertuck** [comment]
> 

Uh, thank you for the style tip!

**Pull request merged**